src/components/freelancer/talent/domainDiag.tsx

```
@@ -121,9 +122,9 @@ const DomainDialog: React.FC<DomainDialogProps> = ({ domains, onSuccess }) => {
   return (
     <Dialog open={open} onOpenChange={setOpen}>
       <DialogTrigger asChild>
-        <Button size="sm">
-          <Plus className="h-4 w-4" /> Add Domain
-        </Button>
+        <div onClick={() => setOpen(true)}>
+          {children}
+        </div>
```

ESLint

[error] Replace `⏎··········{children}⏎········` with `{children}`

(prettier/prettier)